File: NAP II - POO/Python/UFRAcommerce/produto.py
class Produto:
    def __init__(self, nome, preco, volume, peso):
        self.nome = nome
        self.preco = preco
        self.volume = volume
        self.peso = peso

    def __str__(self):
        return f"Produto(nome={self.nome}, preco={self.preco}, volume={self.volume}, peso={self.peso})"

File: NAP II - POO/Java/src/Produto.java
public class Produto {
    private String nome;
    private double preco;
    private double volume;
    private double peso;

    // Construtor padrão
    public Produto(String nome, double preco, double volume, double peso) {
        this.nome = nome;
        this.preco = preco;
        this.volume = volume;
        this.peso = peso;
    }

    // Construtor não padrão
    public Produto(String nome, double preco) {
        this(nome, preco, 0.0, 0.0);
    }

    @Override
    public String toString() {
        return String.format("Produto: %s (Preço: R$ %.2f, Volume: %.2f m³, Peso: %.2f kg)", nome, preco, volume, peso);
    }

    // Getters
    public String getNome() {
        return nome;
    }

    public double getPreco() {
        return preco;
    }

    public double getVolume() {
        return volume;
    }

    public double getPeso() {
        return peso;
    }
}
